qkernel/src/syscalls/sys_proxy.rs

```diff
@@ -184,7 +585,7 @@ pub fn CudaMemcpy(task: &Task, dst: u64, src: u64, count: u64, kind: CudaMemcpyK
             return Ok(1);
         }
         CUDA_MEMCPY_HOST_TO_DEVICE => {
-            // src is the virtual addr
+            
```

Comment:
Remove extra empty line.
Could you have a check on the empty lines? I think there are some places they are so "empty" because of empty lines. I didn't put a comment but may worth you taking a look and remove some empty lines.